.gitignore

```diff
@@ -1,3 +1,7 @@
+.cache
```

Comment:
These belong in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).